src/Fixer/FunctionNotation/PhpdocToParamTypeFixer.php

```diff
@@ -0,0 +1,317 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\FunctionNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\DocBlock\Annotation;
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\VersionSpecification;
+use PhpCsFixer\FixerDefinition\VersionSpecificCodeSample;
+use PhpCsFixer\Preg;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Jan Gantzert <jan@familie-gantzert.de>
+ */
+final class PhpdocToParamTypeFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
+{
+    /**
+     * @var array
+     */
+    private $blacklistFuncNames = [
+        [T_STRING, '__construct'],
+        [T_STRING, '__destruct'],
+        [T_STRING, '__clone'],
+    ];
+
+    /**
+     * @var array
+     */
+    private $skippedTypes = [
+        'mixed' => true,
+        'resource' => true,
+        'null' => true,
+        'static' => true,
+    ];
+
+    /**
+     * @var array
+     */
+    private $versionSpecificTypes = [
+        'void' => 70100,
```

Comment:
I don't know about others, but I like lists like these to be sorted, preferably by key.
If the sorting by version is wanted though, please sort the same versions by key.

_**edit:** same goes for the skipped types and blacklisted functions, please sort them._

Reply:
The sorting is like in the PhpdocToReturnTypeFixer Class. If we want to change the sorting, it should be in all classes or not?

Reply:
In that case ... OK from my side... sorted it by key + value